Sources/ArgumentParser/Parsable Properties/Option.swift

```diff
@@ -138,12 +138,40 @@ extension Option where Value: ExpressibleByArgument {
   ///   - name: A specification for what names are allowed for this flag.
   ///   - parsingStrategy: The behavior to use when looking for this option's value.
   ///   - help: Information about how to use this option.
+  @available(*, deprecated, message: "Use init(wrappedValue:name:parsing:help:completion:) instead.")
   public init(
     wrappedValue: Value,
     name: NameSpecification = .long,
     parsing parsingStrategy: SingleValueParsingStrategy = .next,
-    completion: CompletionKind? = nil,
```

Comment:
This change isn't ABI stable if I recall correctly, but im not sure if that matters for this library. I wonder if you could leave types as Optional with a default of `nil` and use `@_disfavoredOverload` to prefer using your new initializer. 

@natecook1000 to comment on if this is necessary

Reply:
I didn't know about `@_disfavoredOverload`. Indeed, it seems to do the trick (when re-establishing optionals).